restapi/common.go

```diff
@@ -37,7 +37,7 @@ func make_api_object(d *schema.ResourceData, m interface{}) (*api_object, error)
     post_path,
     put_path,
     delete_path,
-    d.Id(),
+    id,
```

Comment:
Argh - thanks for the fix. I have merged this commit into master and will produce a release with the fix!